src/commands/edge_app_server.rs

```diff
@@ -96,39 +102,75 @@ impl Default for Metadata {
 
 #[derive(Debug, Default, Deserialize)]
 struct MockData {
+    metadata: Metadata,
+    settings: HashMap<String, Value>,
+}
+
+#[derive(Debug, Default, Deserialize, Clone)]
+struct MockDataStr {
     metadata: Metadata,
     settings: HashMap<String, String>,
 }
 
+impl MockData {
+    fn new_from_str(mock_data_str: &MockDataStr) -> Self {
+        let settings_val = mock_data_str
```

Comment:
why are we not using serde to deserialize string?

Reply:
Well, that was the failing part. It cannot automatically create the new `MockData` from a string. I needed to create that middle step to fix it. If you have a more idiomatic idea, I'm all ears.

Reply:
What's the issue? It's not enought data for me that it's a failing part. But if some field fails to parse correctly you can always use custom deserializer - take a look at EdgeAppManifest for instance it does that a lot

Reply:
I'd like to learn that custom deserializer. But we need to ship this one quickly. How about implementing it as a follow-up?

Reply:
ok since we need it fast - please create a GH issue and reference it there

Reply:
Will do.

Reply:
But there is no created issue and it's not referenced here :( So it's not resolved

Reply:
https://github.com/Screenly/cli/issues/119 :smile: